src/components/NcAppSidebarTab/NcAppSidebarTab.vue

```diff
@@ -30,8 +30,8 @@
 		:aria-label="isTablistShown() ? undefined : name"
 		:aria-labelledby="isTablistShown() ? `tab-button-${id}` : undefined"
 		class="app-sidebar__tab"
-		tabindex="0"
-		role="tabpanel"
+		:tabindex="isTablistShown() ? 0 : -1"
```

Comment:
```suggestion
		:tabindex="isTablistShown() ? 0 : undefined"
```
Similar to above, the tabindex should not be set at all otherwise it will still be click focusable